social_distance/settings.py

```diff
@@ -85,6 +85,8 @@
         'rest_framework_simplejwt.authentication.JWTAuthentication',
         # 'rest_framework.authentication.SessionAuthentication', # we don't use django built-in session, which imposes csrf
     ],
+    'DEFAULT_PAGINATION_CLASS': 'social_distance.pagination.PageSizePagination',
+    'PAGE_SIZE': 5 # default number of items per page
```

Comment:
I choose the default as 5 here because the spec indicated we should return ~ 5 comments per post
![image](https://user-images.githubusercontent.com/43459469/137433241-502ebdfe-6976-4a03-9d15-473651012b9c.png)

and when calling the endpoint without query parameters then the default will be page=1, size=5